src/parser/Statement.ts

```diff
@@ -466,6 +534,14 @@ export class PrintStatement implements Statement {
         }
         return result;
     }
+    walk(visitor: WalkVisitor, options?: WalkOptions) {
+        if (options?.walkExpressions) {
+            for (let i = 0; i < this.expressions.length; i++) {
+                //TODO the original walk code did an `isExpression()` check before walking. Do we need that?
```

Comment:
I think there were cases where there can be "tokens" in such a list instead of expressions.